### PR TITLE
feat: display only events from venue

### PR DESCRIPTION
**Pull request by gikf**
- [x] I have read [Chapter's contributing guidelines](https://github.com/freeCodeCamp/chapter/blob/main/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `main` branch of Chapter.

---

- Narrows events displayed in the _Venue dashboard_ to events organized at the venue. Instead of displaying all events from the chapter.
- Additionally sorts these events in the descending order by date.

### CODE REVIEW

File: client/src/modules/dashboard/Venues/pages/VenuePage.tsx [hunk 1]
```diff
@@ -38,7 +38,7 @@ export const VenuePage: NextPageWithLayout = () => {
         </ProgressCardContent>
       </Card>
       <EventList
-        title="Organized By The Venue's Chapter"
+        title="Organized At The Venue"
         events={data.venue.chapter.events}
       />
```
Comment (Sboonny) [inline client/src/modules/dashboard/Venues/pages/VenuePage.tsx:43]: What do you think about adding logic to hide `EventList` if `chapter.events.length < 1`?

Comment (gikf) [reply client/src/modules/dashboard/Venues/pages/VenuePage.tsx:43]: Or maybe add some indication of that in the `EventList` component? What do you think?

Comment (Sboonny) [reply client/src/modules/dashboard/Venues/pages/VenuePage.tsx:43]: Good call, `EventList` is used in three pages 👍, and not Venue only.

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Pull request merged**